src/components/spinner/Spinner.tsx

```diff
@@ -135,12 +142,22 @@ const SpinnerStyled = styled.span`
   width: var(--echoes-dimension-size-200);
   border-radius: var(--echoes-border-radius-full);
   vertical-align: text-bottom;
+
+  ${ButtonStyled} & {
+    margin-right: var(--echoes-dimension-size-75);
+
+    background:
+      linear-gradient(0deg, var(--button-spinner-color) 50%, transparent 50% 100%) border-box,
+      linear-gradient(90deg, var(--button-spinner-color) 25%, transparent 75% 100%) border-box;
+  }
```

Comment:
Fixing the build with the missing @emotion/babel-plugin, we can now do that! I think that's the cleanest. It keeps all the styles related to the component encapsulated in a single source of truth.